src/bundle-manifest.ts

```diff
@@ -205,10 +204,9 @@ export function generateSharedDepsMergeStrategy(minEntrypoints?: number):
  * bundle for an application shell.
  */
 export function generateShellMergeStrategy(
-    shell: UrlString, minEntrypoints?: number): BundleStrategy {
-  if (minEntrypoints === undefined) {
-    minEntrypoints = 2;
-  }
+    shell: UrlString, maybeMinEntrypoints?: number): BundleStrategy {
+  const minEntrypoints =
+      maybeMinEntrypoints === undefined ? 2 : maybeMinEntrypoints;
```

Comment:
![Steve McCroskey](https://i.imgur.com/pAvJgUP.jpg)

Looks like I picked the wrong time to start reassigning arguments for defaults.